### DISCUSSION TITLE
is there a similar part in vue-leaflet to LPolyline.js in vue2-leaflet/dist/components/LPolyline.js?

### DISCUSSION THREAD

**Discussion by ali-khoshnevisan** [vue-leaflet/vue-leaflet#253] [Q&A]
hi. i used to be using (__vue_component__)  as an imported file into my project which exists in this path: vue2-leaflet/dist/components/LPolyline.js.
my project is creating a flowing line, using LPolyline and SVG and it works with vue2. (https://github.com/ali-khoshnevisan/vue-leaflet-flowing-line)
recently i migrate to vue3 and i have to using @vue-leaflet/vue-leaflet instead of vue2-leaflet.
but i found nothing like that variable (__vue_component__) in this package's files. also there is not any file like LPolyline.js in this package.
can anyone help me in this case?
is there any file similar that things that i want to use?
is there a chance to add this specificly parts that i want in future developing of this package?

**Comment by mikeu** [comment]
Hi @ali-khoshnevisan , does the [LPolyline component](https://github.com/vue-leaflet/vue-leaflet/blob/master/src/components/LPolyline.vue) help at all? You can find an example of it in action in the [playground demo source](https://github.com/vue-leaflet/vue-leaflet/blob/master/src/playground/views/Polyline.vue) too.
